packages/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react'
+
+import { Divider, Item, List } from './styled'
+
+interface AccordionItemProps {
+  children?: ReactNode
+}
+
+export function AccordionItem({ children }: AccordionItemProps) {
+  return (
+    <Item>
+      {children}
+      <Divider className="accordion-divider" />
```

Comment:
What does the `className` do?

Reply:
In design, I noticed that the last accordion doesn't have divider. so in order to remove it in CSS, I added the classname.
![Screen Shot 2022-09-01 at 2 11 24 PM](https://user-images.githubusercontent.com/93789729/187984181-542f7631-0874-456d-8a67-d36425366668.png)

Reply:
@ryanjdiercks, since we are using styled-components, can we resolve this to avoid having two ways of affecting styles?